components/script/dom/range.rs

```diff
@@ -1204,6 +1226,7 @@ impl WeakRangeVec {
     }
 
     /// Used for steps 2-3. when removing a node.
+    ///
```

Comment:
This empty line makes the formatting in the generated documentation a bit nicer.